examples/trusted-sync/src/main.rs

```diff
@@ -277,12 +279,11 @@ async fn sync(cli: cli::Cli) -> Result<()> {
         metrics::SAFE_L2_HEAD.set(derived);
         metrics::DERIVED_ATTRIBUTES_COUNT.inc();
         println!(
```

Comment:
`info!`?

Reply:
We can, but since the point of `trusted-sync` is to show payloads are validated, I think it makes sense to always print this even without verbosity enabled. Or do you think I should just have the INFO as the default verbosity level?

Reply:
`INFO`, `WARN`, `ERROR` should always be emitted imo

Reply:
`DEBUG`/`TRACE` can be optional

Reply:
Ok I'm fine with this, let's stick to tracing for io :)